### PR TITLE
Added portuguese aside tag explanation

### PR DESCRIPTION
**Pull request by ritagomesdasilva**
Added the portuguese explanation to the element ASIDE.

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [ x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/portuguese/html/elements/aside-tag/index.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
 ---
 title: Aside Tag
-localeTitle: Além de Tag
+localeTitle: Tag 'Aside'
 ---
-## Além de Tag
```
Comment (thefrontendwizard) [inline guide/portuguese/html/elements/aside-tag/index.md]: A explicação está ótima mas o texto precisa seguir o formato recomendado para os guides. Precisa de um título no início e de uma sessão de "Mais Informações" no final com links.

**Review by frontendwizard — DISMISSED** [review]

File: guide/portuguese/html/elements/aside-tag/index.md (hunk 1 above)
Comment (ritagomesdasilva) [reply guide/portuguese/html/elements/aside-tag/index.md]: Repus o título original e acrescentei a secção de Mais informações no final para ficar de acordo com o formato recomendado!

**Comment by RandellDawson** [comment]
@thefrontendwizard Can you validate the new changes made by the OP are good enough to merge?

Thanks

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**